app/models/move.rb

```diff
@@ -35,14 +35,14 @@ class Move < ApplicationRecord
   has_many :hunters, through: :hunters_moves
 
   scope :not_basic, -> { where.not(type: 'Moves::Basic') }
-  scope :with_hunter, (lambda do |hunter_id|
+  scope :with_hunter, ->(hunter_id) do # rubocop:disable Style/Lambda
```

Comment:
I have pretty mixed feelings on whether this is still an improvement 🤔

Reply:
I think it's better. The `end)` had no explanation. Here we're explaining that there is a rule exception.